content/docs/command-reference/repro.md

```diff
@@ -10,14 +10,17 @@ analyzing dependencies and <abbr>outputs</abbr> of the target stages.
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c <path>] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull]
-                 [targets [targets ...]]
+                 [--no-commit] [--downstream] [--pull] [--glob]
+                 [targets [<target> ...]]
 
 positional arguments:
-  targets        Stage or path to dvc.yaml or .dvc file to reproduce. Using -R,
-                 directories to search for stages can also be given.
+  targets        Stage or path to dvc.yaml or .dvc file to reproduce.
+                 If no targets are provided, it is assumed to be the
+                 dvc.yaml present in the current working directory.
 ```
 
+See [\<target\>](#options) for more details.
+
```

Comment:
Hmmm `targets` (plural BTW) is not an option right? It's an argument.

I think we should make a new H3 (`###`) section in the description instead.

Reply:
p.s. Probably before **Parallel stage execution**.

Reply:
Now, `<target>` is a spec, whereas `targets` is a list of `<target>` here. 

I'd suggest to do similar to [git-add](https://git-scm.com/docs/git-add#Documentation/git-add.txt-ltpathspecgt82308203) and keep it in Options, but could do as you suggested. Let me know what you think.

Reply:
OK. But I don't see `<target>` (with `<` `>`) in the help output or any earlier indication that it's some sort of structure with a defined format (a spec). In the Git ref you link to, `<pathspec>` is in the Synopsis. In any case Git cmd refs aren't great inspiration 😋

I think that for now we should stick to a section in the Description @skshetry, but is this a new thing in 2.0? (Having certain specs) Do they have a new/special implementation? Do you know if there's a list of all our specs somewhere in the core repo perhaps? Thanks

Reply:
> but is this a new thing in 2.0? (Having certain specs)

except for the glob, this was there since 1.0. The spec is not just applicable to repro, but most of the commands like commit, checkout, push, pull, etc. Some of them also accept output file path in addition to that spec. But, adding them in all other places is not that feasible and/or important, so only added on `repro`.

> But I don't see <target> (with < >) in the help output

It's hard to add these details on the help output and is something that should be in the manpage.

Reply:
Hmmm.. I'm not sure what you mean by specs, but I still think we should stick to a section in the Description like we already do if nothing changed specially form 1.0 to 2.0.

Reply:
@jorgeorpinel I think @skshetry has a point here. A section in description is not convenient/enough. There should be some simple description for all options. Ideally we should "teach" engine to link to them from the synopsis. I think that's what @skshetry was tryin to imitate, it's just engine is not sophisticated enough to support that yet.

Reply:
Hmmm a different section for **Arguments** then? I guess that could be clearer than explaining them in the description... Not sure, I can try and then we can decide. But it seems like that's not quite in scope for this PR. I'd prefer to use the existing format for now and open a separate issue.